src/arbitrary/constantFrom.ts

```diff
@@ -2,6 +2,21 @@ import { Arbitrary } from '../check/arbitrary/definition/Arbitrary';
 import { convertFromNext } from '../check/arbitrary/definition/Converters';
 import { ConstantArbitrary } from './_internals/ConstantArbitrary';
 
+type Arrayfy<T> = T extends any[] ? T : T[];
+type Elem<T> = T extends any[] ? T[number] : T;
```

Comment:
I usually add this annotation to avoid any risk of unwanted publication of the type. Those ones being fully internals, we can safely use it there.

```suggestion
/** @internal */
type Arrayfy<T> = T extends any[] ? T : T[];
/** @internal */
type Elem<T> = T extends any[] ? T[number] : T;
```